Telnyx Meet/Controllers/VideoMeetRoomViewController.swift

```diff
@@ -614,7 +633,7 @@ class VideoMeetRoomViewController: UIViewController {
 
         if totalParticipants == 1 || totalVideoStreams == 0 {
             // There are no participants or no remote participant is sharing their video.
-            self.publishLocalStream(audio: true, video: true)
+            self.publishLocalStream(audio: false, video: false)
```

Comment:
@guillermobattistel why are we setting `audio` and `video` as `false`?
If there are multiple participants sharing video, we wait for the subscriptions to complete and then we publish local video.
If there are no participants, then there are no subscriptions to wait for, so we publish the local video.